test/testutils/wrappers.go

```diff
@@ -146,6 +146,23 @@ func MakeWorkerPodSpec() corev1.PodSpec {
 	}
 }
 
+func MakeWorkerPodSpecWithInitContainer() corev1.PodSpec {
```

Comment:
Maybe make it a PodWrapper to avoid duplicate codes.

Reply:
Can you link me an example? I think this may be something I tackle in the follow up PR mentioned above.

Reply:
Yes, agree.

Reply:
A simple of this is LeaderWorkerSetWrapper under /test/testutils/wrappers.go or https://github.com/kubernetes/kubernetes/blob/fd2d352d291bc4fb36d51e52b33a6f6849f20f35/pkg/scheduler/testing/wrappers.go#L216

Reply:
Thanks, going to follow up with test code refactoring PR.